src/tech/v3/ml.clj

```diff
@@ -230,6 +243,7 @@ see tech.v3.dataset.modelling/set-inference-target")
      (->> gs-seq
           (pmap #(do-k-fold % loss-fn target-colname ds-seq))
           (sort-by :avg-loss)
-          (take n-result-models))))
+          (take n-result-models)
```

Comment:
This is a unrelated bug fix.
The originalò code did not restrict the number of models returned to `n-result-models'